src/components/Feed/components/TaskUnit/components/SocialUnit/components/MangoPopover/index.js

```diff
@@ -1,42 +1,48 @@
 import * as React from "react";
 import { connect } from "react-redux";
-import { addMango } from "./actions";
-import {
-  Button,
-  Popover,
-  Form,
-} from "react-bootstrap";
 
-class MangoPopup extends React.Component {
+import { Button, Popover } from "react-bootstrap";
+import Typography from "@material-ui/core/Typography";
+import Slider from "@material-ui/core/Slider";
+import { addMangoToTask } from "../../../../../../../../actions/feedActions";
```

Comment:
I set up static file paths so you can use "actions/feedActions" instead

Reply:
I merged it into the project a while back so you should be able to utilize it right away